README.md

```diff
@@ -50,8 +50,9 @@ There are amazing resources that are available for codenewbies to learn and deve
 
 #### React JS repos
 
+- [Awesome React](https://github.com/enaqx/awesome-react) - This repository is a collection of awesome things regarding the React ecosystem. It includes React General Resources, Communities, Online Playgrounds, Tutorials and Tools.
+
 - [Awesome React Resources](https://github.com/brillout/awesome-react-components) - This amazing Github repository has great resources that include the React ecosystem. You can fork or star it will be useful in your next project. The repo includes:React tools, React General Resources, React Community, React Online Playgrounds and React tutorials.
-- [Useful React Resources]() - This amazing Github repository has great resources that include the React ecosystem. You can fork or star it will be useful in your next project. The repo includes:React tools, React General Resources, React Community, React Online Playgrounds and React tutorials.
```

Comment:
I have also removed this resource as It was not pointing to any Github Repositories.